src/main/java/de/craften/plugins/rpgplus/RpgPlus.java

```diff
@@ -62,11 +61,11 @@ public RpgPlus() {
         commandManager = new CustomCommands();
         timerManager = new TimerComponent();
         
-        StorageType storageType = StorageType.valueOf(getConfig().getString("storageType", "YAML").toUpperCase());
+        String storageType = getConfig().getString("storageType", "YAML");

```

Comment:
If the case doesn't matter anyway, there is no need to YELL AT ME! :stuck_out_tongue_winking_eye: